### PR TITLE
Branch manual for Foreman 3.2

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

File: _includes/manuals/3.2/1.1_architecture.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+configurations, initial host configuration files, etc. However, if the
+foreman installation supports unattended installations then other
+operations need to be performed to fully automate this process. The
+smart proxy manages remote services and is generally installed with all
+Foreman installations to manage TFTP, DHCP, DNS, Puppet, Puppet CA,
+Ansible, Salt, and Chef.
+
+## Smart-Proxy
+
+A [Smart-Proxy](manuals/{{page.version}}/index.html#4.3SmartProxies) is located on or near a machine that performs a specific function and helps foreman orchestrate the process of commissioning a new host. Placing the proxy on or near to the actual service will also help reduce latencies in large distributed organizations.
```
Comment (melcorr) [inline _includes/manuals/3.2/1.1_architecture.md]:
```suggestion
A [Smart-Proxy](manuals/{{page.version}}/index.html#4.3SmartProxies) is located on or near a machine that performs a specific function and helps Foreman orchestrate the process of commissioning a new host. Placing the Smart Proxy on or near to the actual service will also help reduce latency in large distributed organizations.
```

File: _includes/manuals/3.2/1.1_architecture.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+## Foreman Architecture
+
+A Foreman installation will always contain a central foreman instance
+that is responsible for providing the Web based GUI, node
+configurations, initial host configuration files, etc. However, if the
+foreman installation supports unattended installations then other
```
Comment (melcorr) [inline _includes/manuals/3.2/1.1_architecture.md]:
```suggestion
Foreman installation supports unattended installations, then other
```

File: _includes/manuals/3.2/1.1_architecture.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+operations need to be performed to fully automate this process. The
+smart proxy manages remote services and is generally installed with all
```
Comment (melcorr) [inline _includes/manuals/3.2/1.1_architecture.md]:
```suggestion
Smart Proxy manages remote services and is generally installed with all
```

File: _includes/manuals/3.2/1.2_release_notes.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+For more information see the [RFC](https://community.theforeman.org/t/drop-require-ssl-and-require-ssl-smart-proxies-settings/26772) and [PR](https://github.com/theforeman/foreman/pull/9021).
+
+### Deprecations
+
+#### Running Foreman on EL7
+
+Foreman 2.1 introduced EL8 support and Katello 4.0 (on Foreman 2.4) followed.
+While it's currently undecided when EL7 support will be dropped exactly, this is an early notice given the considerable number of EL7 deployments.
+For fresh installations, EL8 should considered the preferred target.
```
Comment (melcorr) [inline _includes/manuals/3.2/1.2_release_notes.md]:
```suggestion
For fresh installations, it is advisable to install on EL8.
```

File: _includes/manuals/3.2/1.2_release_notes.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+Existing installations should start thinking about a migration plan.
+
+Note that this is only support to run Foreman and Foreman Proxy themselves on EL7. Managing EL7 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```
Comment (melcorr) [inline _includes/manuals/3.2/1.2_release_notes.md]:
```suggestion
Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on EL7. Managing EL7 client systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```

File: _includes/manuals/3.2/1.2_release_notes.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+#### Running Foreman on Debian 10
+
+Now that Debian 11 is supported, Debian 10 support is deprecated and will be dropped with Foreman 3.4
+
+Note that this is only support to run Foreman and Foreman Proxy themselves on Debian 10. Managing Debia 10 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```
Comment (melcorr) [inline _includes/manuals/3.2/1.2_release_notes.md]:
```suggestion
Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on Debian 10. Managing Debian 10 client systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```

File: _includes/manuals/3.2/2.1_quickstart_installation.md [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+      Initial credentials are admin / 3ekw5xtyXCoXxS29
+  * Foreman Proxy is running at https://theforeman.example.com:8443
+  The full log is at /var/log/foreman-installer/foreman-installer.log
+{% endhighlight %}
+
+#### Next-generation documentation
+
+Over the last year, Foreman community members have been open-sourcing Red Hat documentation to make more comprehensive guides available to Foreman users. At the moment, the following guides have been migrated to a work-in-progress [Foreman and Katello documentation site](https://docs.theforeman.org). This project is not yet complete, but you might find useful information in some of the following guides:
+
+* [Provisioning Guide](https://docs.theforeman.org/master/Provisioning_Guide/index-foreman.html)
```
Comment (melcorr) [inline _includes/manuals/3.2/2.1_quickstart_installation.md:233]: all these links are broken... we'll need a more clever way to handle these..

**Review by melcorr — CHANGES_REQUESTED** [review]
Few small comments I hope might make things clearer.

**Comment by ekohl** [comment]
They look like good suggestions. It would be better to first apply these to nightly and then do the branching. Also, I think some of these need to be applied to other releases as well.

**Comment by melcorr** [comment]
I had intended only to touch the release notes but couldn't resist the architecture section. I'll raise a separate PR for those few changes.

**Comment by upadhyeammit** [comment]
Rebased on @melcorr changes.

**Comment by upadhyeammit** [comment]
while we are at branching manuals I also updated the installer options update in same pr.

**Review by ekohl — APPROVED** [review]
Thanks! CI is still running but :+1: once it's green.

**Review by melcorr — APPROVED** [review]

**Pull request merged**